modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/src/testIntegration/java/com/liferay/headless/admin/list/type/resource/v1_0/test/ListTypeDefinitionResourceTest.java

```diff
@@ -85,6 +103,59 @@ public void testPostListTypeDefinition() throws Exception {
 		_assertListTypeDefinitionNameLocalizedMap(
 			testPostListTypeDefinition_addListTypeDefinition(
 				randomListTypeDefinition));
+
+		// With permissions
```

Comment:
Whenever you add a comment, you need to check whether there were previous comments.
In this case, there were none before your change, so you need to add comments for all the previous scenarios as well. 

in this case the commit message for the previous scenario is:
LPD-37030 Add test for adding a list type definition that does not have a translation for the site default locale

```
public void testPostListTypeDefinition() throws Exception {
super.testPostListTypeDefinition();

// Name
...

// Permissions

```